src/main/java/com/github/hanyaeger/api/engine/entities/entity/sprite/DynamicSpriteEntity.java

```diff
@@ -74,7 +75,20 @@ protected DynamicSpriteEntity(final String resource, final Coordinate2D initialL
      * @param interval the interval milli-seconds
      */
     protected void setAutoCycle(final long interval) {
+        setAutoCycle(interval, -1);
+    }
+
+    /**
+     * Set the interval at which the sprite should be automatically cycled.
+     * The sprite will cycle through one row of sprites, from left to right.
+     *
+     * @param interval the interval milli-seconds
+     * @param row the row to cycle through
+     */
+    protected void setAutoCycle(final long interval, final int row) {
```

Comment:
Although fixing the autocycle to a specific row (and not a random interval or a column) feels a bit restrictive, I guess you're right on this being the most prominent use-case. So lets fo for this approach and see if we get any demand for a even more flexible implementation